.circleci/config.yml

```diff
@@ -273,6 +273,11 @@ workflows:
           target: manifests
           diff-fails: true
           requires: [prepare-env]
+      - go-make:
+          name: mocks
+          target: generate-mocks
```

Comment:
I would not expect mock to self generate/update on their own in the CI

Usually, you should instead FAIL if mocks are not properly generated (compare with git diff and exit)

Reply:
How can I ensure the mocks are updatdate if I don't generate them in the CI and compare the diff' ?

Reply:
The PR as is was not performing any validation or diff, correct?

Let me rephrase then

> I would not expect mock to self generate/update on their own in the CI


I would not expect mock to ONLY self generate/update on their own in the CI WITHOUT performing any follow up assertion.

Reply:
Indeed I misinterpreted the configuration file 😄 

Thanks for clarifying